web/src/main/java/org/openmrs/web/filter/startuperror/StartupErrorFilterModel.java

```diff
@@ -39,8 +43,9 @@ public StartupErrorFilterModel(Throwable t) {
 		errorAtStartup = t;
 		
 		StringWriter sw = new StringWriter();
-		PrintWriter pw = new PrintWriter(sw);
-		t.printStackTrace(pw);
+		//		PrintWriter pw = new PrintWriter(sw);
+		//        t.printStackTrace(pw);
+		log.error(t);
 		stacktrace = sw.toString();
```

Comment:
This should not call log.error(t). It should set stacktrace = ExceptionUtil.getStackTrace(t);.